_projects/roberta_external_source.md

```diff
@@ -2,7 +2,8 @@
 collaborating_projects:
   - coala
   - openroberta
-desc: "Create a loader for projects from remote data sources that can be used from the coala web page in order to provide one click linting for online projects. "
+desc: "Create a loader for projects from remote data sources that can be used from the coala web page in order to
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.